akka-persistence/src/main/scala/akka/persistence/Persistence.scala

```diff
@@ -215,6 +253,34 @@ class Persistence(val system: ExtendedActorSystem) extends Extension {
     system.systemActorOf(pluginActorProps, pluginActorName)
   }
 
+  private def createAdapter(configPath: String): Option[ModelAdapter[Any]] = {
+    val pluginConfig = system.settings.config.getConfig(configPath)
+    if (!pluginConfig.hasPath("adapter")) None
+    else {
+      val adapterClassName = pluginConfig.getString("adapter")
```

Comment:
yes, and that's why I wanted to combine the adapter with the tagger, so that user doesn't have to configure (mostly) same thing twice